monkey/monkey_island/cc/resources/island_mode.py

```diff
@@ -12,30 +13,26 @@
 
 
 class IslandMode(AbstractResource):
-    urls = ["/api/island-mode"]
+    urls = ["/api/island/mode"]
 
     def __init__(self, island_mode_service: IslandModeService):
         self._island_mode_service = island_mode_service
 
-    # API Spec: Instead of POST, this should be PUT
     @jwt_required
-    def post(self):
+    def put(self):
         try:
             body = json.loads(request.data)
             mode = IslandModeEnum(body.get("mode"))
 
             self._island_mode_service.set_mode(mode)
 
-            # TODO: Do any of these returns need a body and make_response? What happens if we just
-            #       return the response code?
-            # API Spec: This should be 204 (NO CONTENT)
-            return make_response({}, 200)
+            return {}, HTTPStatus.NO_CONTENT
         except (AttributeError, json.decoder.JSONDecodeError):
-            return make_response({}, 400)
+            return {}, HTTPStatus.BAD_REQUEST
         except ValueError:
-            return make_response({}, 422)
+            return {}, HTTPStatus.UNPROCESSABLE_ENTITY
 
     @jwt_required
     def get(self):
         island_mode = self._island_mode_service.get_mode()
-        return make_response({"mode": island_mode.value}, 200)
+        return make_response({"mode": island_mode.value}, HTTPStatus.OK)
```

Comment:
While we're working on this (maybe in a follow-up PR), can we just return a string? The dict is redundant; the resource is already called "mode".

```suggestion
        return make_response(island_mode.value, HTTPStatus.OK)
```

Reply:
I think with only a string we can get rid of the `make_response`. I will add another task in the issue.